config/reports.json

```diff
@@ -298,20 +298,20 @@
 				]
 			}
 		},
-		"ttci": {
```

Comment:
As you pointed out, changing the metric ID here also requires renaming the SQL files in the bigquery repo. It also requires renaming the JSON files on the CDN. For example:

- https://cdn.httparchive.org/reports/ttci.json (timeseries)
- https://cdn.httparchive.org/reports/wordpress/ttci.json (lens timeseries)
- https://cdn.httparchive.org/reports/2018_10_15/ttci.json (histogram)
- https://cdn.httparchive.org/reports/wordpress/2018_10_15/ttci.json (lens histogram)

All of those for both metrics, for all 3 lenses, and for all histograms since 2017_06_01.

That's a lot of files. It's doable with a script, but not sure if it's worth the effort. Alternatively, I propose we keep the metric IDs as-is and simply update the `name` property. That would eliminate the need for renaming the JSON and SQL files. How does that sound?

Reply:
I'm totally ok with that, one concern I have is related to understanding and reading the code. Newcomers might be confused by not consistent key names.
But we can do renamings later...

Reply:
SGTM 👍